components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/DefaultOIDCClaimsCallbackHandler.java

```diff
@@ -742,7 +743,12 @@ private boolean isLocalUser(OAuthAuthzReqMessageContext authzReqMessageContext)
         return !authzReqMessageContext.getAuthorizationReqDTO().getUser().isFederatedUser();
     }
 
-    private boolean isMultiValuedAttribute(String claimValue) {
+    private boolean isMultiValuedAttribute(String claimKey, String claimValue) {
+
+        // Address claim is already handled in the required format
```

Comment:
Lets say it could contain a multi attribute sepator but its not a multi valued attribute

Reply:
Fixed